guides/standards-assurance-tables.md

```diff
@@ -0,0 +1,158 @@
+
+### Confidence
+
+The assessor reviews the evidence and uses their professional opinion to indicate a Confidence Score.
+
+The Confidence Score is based on a scale of 0 (no confidence at all) to 14 (high level of confidence), or 'N/A' (where the Objective is not applicable to the system being assessed).
+
+### Delta
+
+The Delta Rating is the resulting 'distance' between the assessed system posture against an Objective and the confidence of the same.
+
+Mathematically, the results are N/A (where the Objective is not applicable to the system being assessed) or 0 to 14 (inc).
+
+A wide delta (higher numerical value) conveys the Objective is not met. A narrow delta (lower numerical value) conveys the Objective is closer to being met.
+
+The Delta Rating is automatically calculated as '14 minus Confidence Score'.
+
+### Further Evidence Required
+
+The assessor indicates what further evidence _types_ in their view are required based on the evidence they have thus far.
+
+The [Further Evidence Required (Help) sheet](https://docs.google.com/spreadsheets/d/1oO9RAZQ6ETsN2iHZb5zcFM0HROliw18MBv89oF-BPeg/edit#gid=679100590) has a calculator which the assessor will use.
+
+This data point was designed as unique numericals as this allows programmatic analysis in the future. This data point and how it is formed is currently under active review.
+
+## Understanding the Objectives, gathering evidence for the assessor
+
+Teams/individuals responsible for the design, creation, implementation, support and maintenance of systems should have viable written evidence (regardless of format) that should be made available to various teams on request, for example, security or to internal audit.
+
+Using the [categories used within the MCSS](https://ministryofjustice.github.io/security-guidance/principles/identify-protect-detect-respond-recover/#identify-protect-detect-respond-recover) as sections some helpful thought questions and documentation expectations are discussed below.
+
+### IDENTIFY
+
+#### Possible documentation
+
+- Team organisation charts
+- Data Privacy Impact Assessments (DPIAs)
+- Information risk management documentation (for example, RMADS)
+- Information flow diagrams
+
+#### Thought questions
+
+- Who is responsible and/or accountable for the the system whether from an operational or budgetary perspective?
+- Who is responsible and/or accountable for the information held inside the system?
+- What security-focused work has been conducted on any suppliers and supplier systems to ensure they are safe for use/integration?
+- Where is the system technically hosted?
+- Where are the places where the system stores data?
+- What are the consequences if the system is unavailable to users or data has been lost/corrupted?
+- What happens for business continuity / disaster recovery if the system is unavailable or data has been lost/corrupted?
```

Comment:
What happens for business continuity / disaster recovery -> What changes - if anything - regarding business continuity / disaster recovery processes or plans